### PR TITLE
Add a tutorial to add frames

### PR DESCRIPTION
**Pull request by Esteban82**
I made a basic example on how to add frames, annotations, gridlines and titles based on the pygmt example.

### CODE REVIEW

**Comment by Esteban82** [comment]
I build the html on my pc and looks well. But I am not sure if I did it well. I see that there ir an issue with Vercel.

**Comment by seisman** [comment]
We should switch from Vercel to ReadTheDocs, see the top post in https://github.com/GenericMappingTools/pygmt/pull/1859.

**Comment by Esteban82** [comment]
I see. I will have a look tomorrow. I hope it will be easy to do.

**Comment by Esteban82** [comment]
Ready for review.

File: docs/tutorials/basics/frames.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Plot Frame
+~~~~~~~~~~
+
+By default, GMT does not add a frame to your plot. For example, we can plot
+the coastlines of the world with a Mercator projection:
+
+
+.. gmtplot::
+
+   gmt begin frames png
```
Comment (joa-quim) [inline docs/tutorials/basics/frames.rst:15]: Good opportunity to introduce one-liners?

Comment (Esteban82) [reply docs/tutorials/basics/frames.rst:15]: I don't think so. This is a tutorial for beginners. I would prefer to keep it as simple as possible. I'll even add the title on a separate line. This way, it's easier for them to copy and paste into their own script."

**Pull request merged**